qa/tasks/cephfs/test_admin.py

```diff
@@ -219,14 +219,55 @@ def is_required(index):
 
             if index % 3 == 0:
                 continue;
-            self.fs.mon_manager.raw_cluster_cmd('fs', 'required_client_features', self.fs.name, 'add', feature)
+            self.fs.required_client_features('add', feature)
             self.assertTrue(is_required(index))
 
             if index % 2 == 0:
                 continue;
-            self.fs.mon_manager.raw_cluster_cmd('fs', 'required_client_features', self.fs.name, 'rm', feature)
+            self.fs.required_client_features('rm', feature)
             self.assertFalse(is_required(index))
 
+    def test_required_client_feature_add_reserved(self):
+        """
+        That `ceph fs required_client_features X add reserved` fails.
+        """
+
+        try:
+            c = ["fs", "required_client_features", self.fs.name, 'add', 'reserved']
+            p = self.mon_manager.run_cluster_cmd(args=c, stdout=StringIO(), stderr=StringIO(), wait=False)
+            p.wait()
+        except CommandFailedError:
+            self.assertIn('Invalid feature name', p.stderr.getvalue())
+        else:
+            self.fail("reserved keyword should be invalid")
```

Comment:
nit: something like following would be simpler and shorter IMO -

```suggestion
        p = self.fs.required_client_features(args='add reserved', check_status=False)
       self.assertIn('Invalid feature name', p.stderr.getvalue())
```

Reply:
Great suggestion, thanks!